frontend/src/components/app/app.tsx

```diff
@@ -7,10 +7,15 @@ import { Link, Route, Routes } from 'components/common/common';
 import logo from 'assets/img/logo.svg';
 import Ork from 'components/okrs/okrs';
 import Profile from 'components/profile/profile';
+import { NotFound } from 'components/notFound/NotFound';
 
 const App: React.FC = () => {
   const { pathname } = useLocation();
 
+  const isIncludePath = Object.values(AppRoute).includes(pathname as AppRoute);
```

Comment:
redundant logic, you can use standard router flow - https://www.w3schools.com/react/react_router.asp